exchanges/btcmarkets/btcmarkets_test.go

```diff
@@ -500,3 +503,14 @@ func TestGetActiveOrders(t *testing.T) {
 		t.Fatal(err)
 	}
 }
+
+func TestWsOrderNotification(t *testing.T) {
```

Comment:
Please add ws test coverage for heatrbeat, ticker, orderbook, trade plus the already existing orderChange / fundChange request